components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/service/api/DeviceOrganizationService.java

```diff
@@ -0,0 +1,670 @@
+package org.wso2.carbon.device.mgt.jaxrs.service.api;
```

Comment:
@n-jay We need to add licence header.

Reply:
Fixed d475b8c7afaf20815e85040a0a3a8b15e3a4f320